subprojects/core/src/main/java/org/gradle/api/internal/file/DefaultProjectLayout.java

```diff
@@ -96,7 +96,9 @@ public FileCollection files(Object... paths) {
 
     @Override
     public ConfigurableFileCollection configurableFiles(Object... files) {
-        DeprecationLogger.deprecateMethod("ProjectLayout.configurableFiles()").replaceWith("ObjectFactory.fileCollection()").nagUser();
+        DeprecationLogger.deprecateMethod("ProjectLayout.configurableFiles()").replaceWith("ObjectFactory.fileCollection()")
+            .withJavadoc(ProjectLayout.class)
```

Comment:
We should add a section for this to the upgrade guide if there isn't one vs using javadoc